pkg/build/generated/informers/externalversions/build/v1/buildconfig.go

```diff
@@ -0,0 +1,52 @@
+// This file was automatically generated by informer-gen
```

Comment:
you also seem to have fallen victim to the unnecessary buildconfig informer generation.  delete all these generated files.